ansible/ansible_collections/agentic/agent_configs/plugins/modules/bun_global.py

```
@@ -1,11 +1,135 @@
+control the requested package and the Bun installation paths.
+
+Example task::
+
+    - name: Install Biome with Bun
+      agentic.agent_configs.bun_global:
+        name: '@biomejs/biome'
+        version: 2.3.8
+        state: present
+"""
+
 #!/usr/bin/python
```

Ruff (0.15.11)

[warning] Shebang should be at the beginning of the file

(EXE005)